src/components/KonvaNode.js

```diff
@@ -109,7 +109,7 @@ export default function (nameNode) {
 
       const isContainer = CONTAINERS[nameNode];
       return isContainer
-        ? () => h('template', {}, slots.default?.())
+        ? () => h('v-container', {}, slots.default?.())
```

Comment:
What if the user installs vue-konva with an alternative prefix, wouldn't this fail?
https://github.com/konvajs/vue-konva/blob/0f3a32932dff45973b37d690eeb39d38b5418c92/src/index.js#L47

Reply:
@scythianfeather good point, fixed that